govcd/org_test.go

```diff
@@ -120,7 +128,14 @@ func (vcd *TestVCD) Test_UpdateOrg(check *C) {
 	err = org.Delete(true, true)
 	check.Assert(err, IsNil)
 	// Check if org still exists
-	org, err = GetAdminOrgByName(vcd.client, TestUpdateOrg)
+	for i := 0; i < 30; i++ {
```

Comment:
I suggest replacing this repeated loop with a function named `doesOrgExist`, which may also be reused by other tests.

Reply:
Fixed ;-)